packages/styles/src/badges/README.md

```diff
@@ -0,0 +1,86 @@
+# Badges
+
+Badges are useful for creating labels and tags. They are commonly used also for notifications.
+
+## Default
+
+```html
+<span class="as-badge"> Basic badge</span>
+```
+
+## Colors
+
+You need to add a background color modifier to change the badge color. There are five specific color for badges:
+- `as-bg--badge-gray`
+- `as-bg--badge-blue`
+- `as-bg--badge-pink`
+- `as-bg--badge-green`
+- `as-bg--badge-yellow`
+
+You can also use any other background utility class, like `as-bg--warning`.
+
+```html
+<span class="as-badge as-bg--badge-gray"> Gray badge</span>
+<span class="as-badge as-bg--badge-blue"> Blue badge</span>
+<span class="as-badge as-bg--badge-pink"> Pink badge</span>
+<span class="as-badge as-bg--badge-green"> Green badge</span>
+<span class="as-badge as-bg--badge-yellow"> Yellow badge</span>
+<span class="as-badge as-bg--warning">Warning badge</span>
+<span class="as-badge as-bg--as-complementary">Complementary badge</span>
```

Comment:
This color name is wrong, it should be `as-bg--complementary`

Reply:
👍